### PR TITLE
TA-1313: Test implementation , refactor

### PR DESCRIPTION
**Pull request by EvgeniiNikonets**

### CODE REVIEW

**Comment by EvgeniiNikonets** [comment]
Check only rate_limit_with_global_filter_with_acl_with_flow_control.json another is refactor

File: example_suites/rate_limit_with_global_filter_with_acl_with_flow_control.json [hunk 1]
```diff
@@ -0,0 +1,641 @@
+{
+  "Name": "Rate limit - with Global filter- with ACL- with Flow control TA-1313",
+  "Skip": false,
+  "Tests": [
```
Comment (olgasobolevsky) [inline example_suites/rate_limit_with_global_filter_with_acl_with_flow_control.json:4]: Please reorganize the test:
1. Create rate limit - rl1 - as described in first step except Global:false
2. Create rate limit - rl2 - as described in first step (Global:true) and the changes from step 3 (Include:my-test; number of events:0)
3. Create Global filter with entry:Path:/test-gf
4. Create flow control policy
5.  Create security policy:test-sp with path mappings:
 name: test-gf; match: /test-gf ; limits[rl1]
 name: test-acl; match: /test-acl; acl:acl-deny-all; limits[rl1]
6. Create server group with security policy:test-sp
7. Publish
8. Send requests for step 1 to path: /test-gf
9. Wait 20 sec
10. Send request for step 2 to path: /test-acl
11. Send request for step 3
12. Assert logs

File: example_suites/rate_limit_with_global_filter_with_acl_with_flow_control.json [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+          "ID": "publish-changes",
+          "Skip": false
+        },
+        {
+          "ID": "wait-for-publish",
+          "Skip": false
+        },
+        {
+          "Name": "Send Traffic using GET method to path /test-gf 1",
+          "Skip": false,
```
Comment (olgasobolevsky) [inline example_suites/rate_limit_with_global_filter_with_acl_with_flow_control.json:370]: Step 1: all 3 requests to /test-gf should be blocked with status:470

File: example_suites/rate_limit_with_global_filter_with_acl_with_flow_control.json [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+              },
+              {
+                "Key": "active",
+                "Value": "true"
+              },
+              {
+                "Key": "action",
+                "Value": "\"global-filter block\""
+              },
+              {
```
Comment (olgasobolevsky) [inline example_suites/rate_limit_with_global_filter_with_acl_with_flow_control.json:209]: Action is by id, not name:
{
                "Key": "action",
                "Value": "\"action-global-filter-block\""
              }
Screen Shot 2024-01-25 at 10 44 51

**Review by olgasobolevsky — APPROVED** [review]

**Pull request merged**